src/esm_runscripts/oasis.py

```diff
@@ -465,9 +465,24 @@ def add_restart_files(self, restart_file, fconfig):
                 # folder of the parent experiment to be branched off from:
                 glob_search_file = f"{config['ini_restart_dir']}{restart_file}_????????-{config['ini_restart_date'].year}{config['ini_restart_date'].month:02}{config['ini_restart_date'].day:02}"
                 branchoff_restart_file = glob.glob(glob_search_file)
-                # Note: What if there are more than one found?
+                branchoff_restart_file.sort()
                 if branchoff_restart_file:
-                    branchoff_restart_file = os.path.basename(branchoff_restart_file[0])
+                    # Note: What if there are more than one found? Let the user decide which one to take.
+                    if len(branchoff_restart_file) == 1:
+                        branchoff_restart_file = os.path.basename(branchoff_restart_file[0])
+                    else:
+                        if not gconfig["isinteractive"]:
```

Comment:
Shouldn't this be the other way around?
```suggestion
                        if gconfig["isinteractive"]:
```

Reply:
I thought so too. But (for a reason I do not understand yet) this will be executed twice, actually four time (for each restart file rstas.nc and rstos.nc) and than a second time which will overwrite everything again. At this second round gconfig["isinteractive"] is False. Thats why I ask here for 'if not'.

Reply:
It should be `if gconfig["isinteractive"]` because we want to avoid opening questionairs while ESM-Tools runs in the background on a computing node, that will stall the job. If in the second round `gconfig["isinteractive"] is False` as you said @nwieters, then changing this should not be a problem right?